api/operator.go

```diff
@@ -79,3 +99,61 @@ func (op *Operator) RaftRemovePeerByAddress(address string, q *WriteOptions) err
 	resp.Body.Close()
 	return nil
 }
+
+// KeyringInstall is used to install a new gossip encryption key into the cluster
+func (op *Operator) KeyringInstall(key string) error {
```

Comment:
Should these functions take an argument for override the ACL token used? Some of the other api functions do, but some don't so I wasn't sure.